sacred/observers/file_storage.py

```diff
@@ -109,7 +109,8 @@ def started_event(self, ex_info, command, host_info, start_time, config,
                         raise
         else:
             self.dir = os.path.join(self.basedir, str(_id))
-            os.mkdir(self.dir)
+            if not os.path.exists(self.dir):
+                os.mkdir(self.dir)
```

Comment:
This is an unrelated change isn't it? 
Which problem is this fixing? Wouldn't we want the FileStorageObserver to complain if it is overwriting an existing entry?

Reply:
Yep, didn't mean to submit this. Currently when queuing an experiment that has loaded both a mongo observer and a fileobserver (e.g. for reliability), the fileobserver creates a new folder. If the queued run is then executed on the same host, the fileobserver throws an error because the folder already exists.